### PR TITLE
[MCPE] Skin Fix

### PR DESCRIPTION
**Pull request by colinrgodsey**

### CODE REVIEW

File: src/protocolsupport/protocol/packet/middleimpl/clientbound/play/v_pe/PlayerListSetEntry.java [hunk 1]
```diff
@@ -56,21 +57,25 @@ public PlayerListSetEntry(ConnectionImpl connection) {
 				PESkinsProvider skinprovider = PESkinsProviderSPI.getProvider();
 				ClientBoundPacketData serializer = ClientBoundPacketData.create(PEPacketIDs.PLAYER_INFO);
 				serializer.writeByte(0);
-				VarNumberSerializer.writeVarInt(serializer, infos.size());
+				VarNumberSerializer.writeVarInt(serializer, infos.size() - 1);
```
Comment (7kasper) [inline src/protocolsupport/protocol/packet/middleimpl/clientbound/play/v_pe/PlayerListSetEntry.java]: Woah.. What if the list is send by plugin etc, and contains not the player itself?
To fix maybe employ fixed varint write and store writerindex, then overwrite with fixed varint with correct length.

Comment (7kasper) [reply src/protocolsupport/protocol/packet/middleimpl/clientbound/play/v_pe/PlayerListSetEntry.java]: Kind of like the implementation in EntityMetadata middle packet.

File: src/protocolsupport/protocol/packet/middleimpl/clientbound/play/v_pe/PlayerListSetEntry.java [hunk 2]
```diff
@@ -53,23 +53,28 @@ public PlayerListSetEntry(ConnectionImpl connection) {
 		switch (action) {
 			case ADD: {
 				PESkinsProvider skinprovider = PESkinsProviderSPI.getProvider();
+				boolean containsOurPlayer = infos.containsKey(connection.getPlayer().getUniqueId());
```
[the remark this answers is not in the document]
Comment (colinrgodsey) [reply src/protocolsupport/protocol/packet/middleimpl/clientbound/play/v_pe/PlayerListSetEntry.java]: actually did it right in my update lol

**Comment by colinrgodsey** [comment]
think this should be ready to go unless there's more review. I have another branch ready that builds off of the 1.8 + etc2 branches combined

**Comment by 7kasper** [comment]
Indeed! Awesome.

**Pull request merged**